sui_programmability/examples/frenemies/sources/leaderboard.move

```diff
@@ -197,7 +197,8 @@ module frenemies::leaderboard {
             };
         };
         // new top score!
-        vector::insert(v, new_score, 0)
+        vector::insert(v, new_score, 0);
+        vector::pop_back(v)
```

Comment:
This unfortunately won't work in all cases.  Consider what happens when the leaderboard is initially empty -- this code will end up inserting a score and then immediately popping it.

Reply:
So what's the right fix here?

Reply:
Fix has landed in #8206 (https://github.com/MystenLabs/sui/pull/8206/commits/090ef14fffcd62d17537401ddb117e73f5f4efb3) (which is a rewrite of this function)